src/libcore/tests/test_xml.py

```diff
@@ -320,31 +320,33 @@ def test21_xml_save_plugin(variant_scalar_rgb):
             "radius" : 10.0,
         }
         })
-    s2 = xml.load_file(abs_path)
+    s2 = xml.load_file(filepath)
     assert str(s1) == str(s2)
+    rmtree(os.path.split(filepath)[0])
 
 @fresolver_append_path
 def test22_xml_missing_type(variant_scalar_rgb):
-    from mitsuba.python.dict_to_xml import WriteXML
+    from mitsuba.python.dict_to_xml import dict_to_xml
```

Comment:
Could we rename the python file?
`dict_to_xml.py -> xml.py`

Reply:
Sure :)